From 665f7bb525b2a800257af5d3d83b796e6cd9ddbe Mon Sep 17 00:00:00 2001
From: "Dmitry V. Levin" <ldv@strace.io>
Date: Sun, 9 Jan 2022 20:00:00 +0000
Subject: [PATCH 4/5] xlat: add AUDIT_ARCH_LOONGARCH* constants

* src/xlat/audit_arch.in (AUDIT_ARCH_LOONGARCH32,
AUDIT_ARCH_LOONGARCH64): New constants.

Link: https://lore.kernel.org/lkml/20211013063656.3084555-4-chenhuacai@loongson.cn/
---
 src/xlat/audit_arch.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/xlat/audit_arch.in b/src/xlat/audit_arch.in
index 63d3fe52b..0760eb798 100644
--- a/src/xlat/audit_arch.in
+++ b/src/xlat/audit_arch.in
@@ -25,6 +25,8 @@ AUDIT_ARCH_H8300	(EM_H8_300) /* Removed in v3.13-rc1~130^2~2 */
 AUDIT_ARCH_HEXAGON	(EM_HEXAGON)
 AUDIT_ARCH_I386		(EM_386|__AUDIT_ARCH_LE)
 AUDIT_ARCH_IA64		(EM_IA_64|__AUDIT_ARCH_64BIT|__AUDIT_ARCH_LE)
+AUDIT_ARCH_LOONGARCH32	(EM_LOONGARCH|__AUDIT_ARCH_LE)
+AUDIT_ARCH_LOONGARCH64	(EM_LOONGARCH|__AUDIT_ARCH_64BIT|__AUDIT_ARCH_LE)
 AUDIT_ARCH_M32R		(EM_M32R)
 AUDIT_ARCH_M68K		(EM_68K)
 AUDIT_ARCH_MICROBLAZE	(EM_MICROBLAZE)
-- 
2.34.1

